app/views/admin/items/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<head>
+	<meta charset="utf-8">
+	<title>Admin's Shop</title>
+</head>
+<body>
```

Comment:
Let's start removing these from the view since they're already being rendered in `application.html.erb`